src/utils/cartModifiers.ts

```diff
@@ -0,0 +1,24 @@
+import { Cart } from '../contexts/Cart';
```

Comment:
What if instead of having it in utils, we have this inside the cart context? So instead of having cart and setCart there as valies, we could have cart and all the ways of modifying a cart as actions: changeQuantity, removeItem, addItem etc.

@Si3rr4wow would this approach be better?